files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
+# invert this setting while it's "locked".  Confusingly, this setting
+# is updated every time you exit the game, based on whether the caps
+# lock key was on or off at the time you exited.
+always run = false
 
-debug = false
+# This floating point setting controls the camera/mouse sensitivity
+# when in "look mode".  The default sensitivity is 1.0, with smaller
+# values requiring more mouse movement, and larger values requiring
+# less.  This setting is multiplicative in magnitude.  This setting
+# does not affect mouse speed in GUI mode.
+camera sensitivity = 1.0
 
-[HUD]
-crosshair = true
+# This floating point setting controls the vertical camera/mouse
+# sensitivity relative to the horizontal sensitivity (see "camera
+# sensitivity") above.  It is multiplicative with the previous
+# setting, meaning that it should remain set at 1.0 unless the player
+# desires to have different sensitivities in the two axes.
+camera y multiplier = 1.0
 
-[Objects]
-shaders = true
+# OpenMW will capture control of the cursor if this boolean setting is
+# true.  In "look mode", OpenMW will capture the cursor regardless of
+# the value of this setting (since the cursor/crosshair is always
+# centered in the OpenMW window).  However, in GUI mode, this setting
+# determines the behavior when the cursor is moved outside the OpenMW
+# window.  If true, the cursor movement stops at the edge of the
+# window preventing access to other applications.  If false, the
+# cursor is allowed to move freely on the desktop.
+#
+# This setting does not apply to the screen where escape has been
+# pressed, where the cursor is never captured.  Regardless of this
+# setting "Alt-Tab" or some other operating system dependent key
+# sequence can be used to allow the operating system to regain control
+# of the mouse cursor.  This setting interacts with the "minimize on
+# focus loss" setting by affecting what counts as a focus loss.
+# Specifically on a two-screen configuration it may be more convenient
+# to access the second screen with setting disabled.
+grab cursor = true
+
+# Invert the vertical axis while in "look mode".  If this setting is
+# true, moving the mouse away from the player will look down, while
+# moving it towards the player will look up.  This setting does not
+# affect cursor movement in GUI mode.
+invert y axis = false
+
+# This boolean setting causes the behavior of the sneak key (Ctrl by
+# default) to toggle sneaking on and off rather than requiring the key
+# to be held while sneaking.  Players that spend significant time
+# sneaking may find the character easier to control with this option
+# enabled.
+toggle sneak = false
+
+# This setting continues to be loaded and saved, but has no known
+# effect.  Presumably it and a related but also removed option named
+# "ui y sensitivity" used to control mouse sensitivity while in GUI
+# mode.  The default value is 1.0.
+ui sensitivity = 1.0
```

Comment:
This setting is no longer used because we're using hardware cursor movement. I must have forgotten to remove it when hardware cursors were implemented.

With hardware cursors, the only way to control the cursor's sensitivity is via the operating system's mouse settings. Having the game access and change the operating system settings would be a) complicated to implement and b) potentially inadvisable since we'd be left with the application specified settings if the application crashed.

Removed in master.